test/TritonGPU/remove-redundant-convert.mlir

```diff
@@ -0,0 +1,80 @@
+// RUN: triton-opt %s -split-input-file -tritongpu-remove-layout-conversions 2>&1 | FileCheck %s
+// CHECK-LABEL: remove_convert
+// CHECK-NOT: triton_gpu.convert_layout [[REGISTER:%[0-9]+]]#[[IDX:[0-9]+]] : tensor<1x512xf32
+#blocked = #triton_gpu.blocked<{sizePerThread = [1, 1], threadsPerWarp = [1, 32], warpsPerCTA = [1, 8], order = [1, 0]}>
```

Comment:
any chance this can be reduced a bit and moved to combine.mlir?

Reply:
I think we can remove the test case as combine.mlir actually has a test case that is a reduced form of this.